### ISSUE TITLE
Initially open diary on *next* day as appropriate

### ISSUE THREAD

**Issue by h-h-h-h** [davidhealey/waistline#469]
As I already wrote in another issue:

> I sometimes make entries for the following day, because of fluctuating sleeping times. You should generally take into account that shift workers use the app, leading to such usage.

When I have following-day entries and the app opens anew, it does so on the *current* calendar day. (I've set it to open with the diary view.)

I'm not sure what's it about with the app's ability to create far-future entries. But, since this is possible, it doesn't seem to make sense to always open on the *last* day in the stream of time that has entries. (What about forbidding far-future entries, though?)

So, I propose to always open on the next calendar day instead of the current one, whenever it has entries.

**Comment by davidhealey** [comment]
I often enter several days worth of entries if I've planned certain meals throughout the week, but I still use the current day to add other items as I go along.

**Comment by h-h-h-h** [comment]
I see. So it definitely would need to be an option to open on the next day as appropriate, or it would annoy you.

**Comment by davidhealey** [comment]
If anyone has any interest in this then comment below. This is the sort of thing that won't be implemented unless there is considerable demand for it.

**Comment by h-h-h-h** [comment]
Or someone does it themselves, right?

**Comment by EmilJunker** [comment]
> Or someone does it themselves, right?

Don't hold your breath on waiting for me to implement this one. I mostly tend to implement things that I'm interested in for my own usage. And this feature isn't one of those.

@h-h-h-h But of course you could implement it yourself.

**Comment by davidhealey** [comment]
> Or someone does it themselves, right?

Not necessarily. Anyone is free to edit the source code and add whatever features they want, but that doesn't mean I will merge those features into my source code. I hate feature creep and Waistline is already becoming far more bloated than I ever intended so I'm reluctant to add any niche features unless it useful to a large proportion of users or to me personally.
